Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -285,15 +285,13 @@ class TabLocationView: UIView {
     }
 
     fileprivate func updateTextWithURL() {
-        if let host = url?.host, AppConstants.MOZ_PUNYCODE {
-            urlTextField.text = url?.absoluteString.replacingOccurrences(of: host, with: host.asciiHostToUTF8())
-        } else {
-            urlTextField.text = url?.absoluteString
-        }
-        // remove https:// (the scheme) from the url when displaying
-        if let scheme = url?.scheme, let range = url?.absoluteString.range(of: "\(scheme)://") {
-            urlTextField.text = url?.absoluteString.replacingCharacters(in: range, with: "")
-        }
+        //Commented code below as AppConstants.MOZ_PUNYCODE is false for release always.
+//        if let host = url?.host, AppConstants.MOZ_PUNYCODE {
```

Comment:
- [ ] Completely remove this code
- [ ] Additionally, delete the `MOZ_PUNYCODE` constant.